Makefile

```diff
@@ -165,7 +165,12 @@ docker-build-manager: manager
 	docker build --build-arg TARGETARCH=${OS_ARCH} -t ${MANAGER_IMAGE} -f bin/manager/Dockerfile ./bin/manager/
 	docker save ${MANAGER_IMAGE} -o ./bin/manager/manager.tar.gz
 
-docker-build-injector: injector
+docker-build-ebpf:
```

Comment:
why don't we need a `lima-build-ebpf` or a `ebpf`? (see lines 49-51, for injector:)

Reply:
Good question!

> Context

It is not possible to build an eBPF program on MacOs due to the eBPF linux dependencies. I have created an `ebpf-builder-${ARCH}` docker image responsible to compile eBPF programs. This is why a single step (docker-build-ebpf) exists to build the `ebpf-builder` docker image, then runs this image to compile eBPF programs and moved eBPF binaries into the `bin/injector/ebpf` folder. The folder `bin/injector/ebpf`is copied into the injector image to be used into injections. It explains why the `docker-build-injector` is dependents to the `docker-build-ebpf`. 

> why don't we need a lima-build-ebpf

eBPF programs don't have a dedicated docker image to run them. They are executed into the injector docker image. The `ebpf-builder-${ARCH}` docker image generated by the `docker-build-ebpf` step is only used to build eBPF programs.

> why don't we need a ebpf

It is not possible to build an eBPF program on MacOS due to the eBPF Linux dependencies. It is possible to create an `ebpf` step that run the `ebpf-builder-${ARCH}` docker image to build eBPF programs, but the `ebpf` step will have the docker-build-ebpf step in dependency requirement. 

I am open to other solutions to manage eBPF program 👍

Reply:
Thanks for the explanation!